FE-640 Reward-boosts (#80)
WeatherXM/docs: docs/rewards/reward-boosts.mdx

---
id: reward-boosts

> title: Rewards Boosts

Reviewer: Rewards -> Reward

> sidebar_label: Rewards Boosts

Reviewer: Rewards -> Reward

> slug: /project/rewards-boosts

Reviewer: rewards-boosts -> reward-boosts

---

import useBaseUrl from "@docusaurus/useBaseUrl";

> ## Overview

Reviewer: Let's use `###` please for better structure below.

> In addition to the **Base Rewards**, a device already gets for contributing to weather data during the regular daily distribution. Learn more about the daily rewards [Reward Mechanism (RM) ](/reward-mechanism).

Reviewer: I think a sentence is missing here before "In addition" e.g. "Reward boosts are rewards......"

> **Reward boosts** are additional rewards given to stations that contribute weather data during daily distribution,in addition to the [**base rewards**](/reward-mechanism#daily-rewards-calculation-process). Learn more about the daily rewards in the [Reward Mechanism (RM) ](/reward-mechanism) section.

Reviewer: daily distribution,in addition to the -----> daily distribution, in addition to the

**Reward Boosts** rewards devices with extra $WXM tokens for meeting specific criteria. 

> So far the following **Reward Boosts** are implemented:

Reviewer: Hmmm not true, Beta Rewards is almost implemented, but Cell Boost is **not**, it's just the next boost down in the pipeline. So let's not say "implemented" here as it's a wrong term.

> **Reward Boosts** rewards stations with extra $WXM tokens for meeting specific criteria. 

Reviewer: rewards stations ---> reward stations

Reviewer: Fix this also @ipanagiv please!

### Beta Rewards

> According to the [Token Allocation table](https://weatherxm.network/docs/wxm-token.html#token-allocation) 3M $WXM are reserved to reward stations that participated to the network before the Token Launch Day ("Beta period").

Reviewer: a comma is needed before the "3M $WXM"

Beta rewards distribution will last 425 days, with Day 1 being the Token Launch day.

> Calculation:

Reviewer: Calculation --> #### Calculation

The following calculations will take place for each station, the day before the Token Launch:

> `RewardableBetaHours[s]`.

Reviewer: Instead of a dot at the end please use `:`

This is the total number of hours the station provided valid data during the Beta period.
A rewardable hour is an hour during which the station sent at least one valid weather data point.

> `StationBetaRewards[s] = 3000000 * RewardableBetaHours[s]/Sum(RewardableBetaHours of all stations)`

Reviewer: Add at the end please a `:`

> This is the percentage of the 3m tokens that correspond to the rewardable hours the station provided.

Reviewer: 3m --> 3M

`DailyBetaRewards[s] = StationBetaRewards[s]/365`

> This is the beta rewards allocated daily to this station.

Reviewer: Please move those in a new line and do the following:

`DailyBetaRewards[s] = StationBetaRewards[s]/365`: This is the beta rewards allocated daily to this station.

:::info

> Stations must have a wallet defined to be part of the Beta Rewards allocation.

Reviewer: Let's rephrase it to:

> Each user who has a station eligible for beta rewards, must have a wallet defined to be part of the Beta Rewards allocation.

And hyperlink the "wallet" to the "Add/edit wallet address" section in our docs please.

:::

:::note
The Beta Rewards allocation is managed by the [WeatherXM Network](https://weatherxm.network).
:::

### Cell Boost

Cell Boost is a mechanism that can be used to incentivize waether stations to be deployed on a specific cell. 

> #### Examples of reward programs that may be implemented using **Rewards Boost**, in the near future are:

Reviewer: Let's rephrase it to:

> Examples of reward boosts that may be implemented in the near future are:

And instead of `####` use `###`

> - As part of the &&Beta Rewards** program, the first 100 devices that join the network will receive tokens.

Reviewer: &&Beta Rewards** --> **Beta Rewards**

- Backfill of rewards that were not allocated to certain devices due to malfunctions in the Quality of Data (QoD) mechanism](/project/quality-of-data) or [Proof of Location (PoL) mechanism](/project/proof-of-location).
- Public Goods Funding (PGF) for devices that are deployed and contribute to the network in a humanitarian way, such as providing data in an underserved area or during a specific time. 
- Devices that contribute to the network during a specific event, such as a natural disaster, will receive a boost in rewards.
- Demand driven rewards, a third party who needs data from specific cells.

:::tip

> The **Rewards Boost** is a mechanism that can be used to incentivize specific behavior from devices. It is a flexible mechanism that can be used to incentivize any behavior that is beneficial to the network.

Reviewer: Rewards Boost -> Reward Boosts

:::

:::info

> All the above are **examples** of reward programs that may be implemented using the Rewards Boost mechanism. 

Reviewer: Let's rephrase it to:

> All the above are **examples** of reward boosts that may be implemented.

---
id: reward-boosts
title: Reward Boosts
sidebar_label: Reward Boosts
slug: /project/reward-boosts
---

import useBaseUrl from "@docusaurus/useBaseUrl";

### Overview

> **Reward boosts** are additional rewards given to devices that contribute weather data during daily distribution, added to their **base rewards**. Learn more about the daily rewards at the [Reward Mechanism (RM) ](/reward-mechanism) section.

Author: added.

**Reward Boosts** rewards devices with extra $WXM tokens for meeting specific criteria. 

So far the following **Reward Boosts** are sheduled to be implemented:

### Beta Rewards

> According to the [Token Allocation table](https://weatherxm.network/docs/wxm-token.html#token-allocation), 3M $WXM are reserved to reward stations that participated to the network before the Token Launch Day ("Beta period").

Reviewer 2: ```suggestion
According to the [Token Allocation table](https://weatherxm.network/docs/wxm-token.html#token-allocation), 3M $WXM are reserved to reward stations that participated to the network before the Mainnet Launch Day ("Beta period").
```

Beta rewards distribution will last 425 days, with Day 1 being the Token Launch day.

#### Calculation:

> The following calculations will take place for each station, the day before the Token Launch:

Reviewer 2: ```suggestion
The following calculations will take place for each station, the day before the Mainnet Launch:
```

`RewardableBetaHours[s]`:
This is the total number of hours the station provided valid data during the Beta period.
A rewardable hour is an hour during which the station sent at least one valid weather data point.

> `StationBetaRewards[s] = 3000000 * RewardableBetaHours[s]/Sum(RewardableBetaHours of all stations)`:

Reviewer 2: This syntax is somewhat confusing. I strongly suggest we go with Math formula syntax, as described here:
https://github.blog/2022-05-19-math-support-in-markdown/

The content could be take - and adapted - from here our internal outline wiki.

This is the percentage of the 3M tokens that correspond to the rewardable hours the station provided.

`DailyBetaRewards[s] = StationBetaRewards[s]/365`: This is the beta rewards allocated daily to this station.

:::info
Each user who has a station eligible for beta rewards, must have a wallet defined to be part of the Beta Rewards allocation. Learn [how to add or edit a wallet address](/wallet/add-edit-wallet-address).
:::

:::note
The Beta Rewards allocation is managed by the [WeatherXM Network](https://weatherxm.network).
:::

### Cell Boost

> Cell Boost is a mechanism that can be used to incentivize waether stations to be deployed on a specific cell. 

Reviewer 2: ```suggestion
Cell Boost is a mechanism that can be used to incentivize weather station deployments on a specific cells / areas of interest.
```

### Examples of reward programs that may be implemented using **Rewards Boost**, in the near future are:

> - As part of the **Beta Rewards** program, the first 100 devices that join the network will receive tokens.

Reviewer 2: This is just an idea, not to be publicly communicated.
```suggestion
```

- Backfill of rewards that were not allocated to certain devices due to malfunctions in the [Quality of Data (QoD) mechanism](/project/quality-of-data) or [Proof of Location (PoL) mechanism](/project/proof-of-location).
- Public Goods Funding, for devices that are deployed and contribute to the network in a humanitarian way, such as providing data in an underserved area or during a specific time. 

> - Devices that contribute to the network during a specific event, such as a natural disaster, will receive a boost in rewards.

Reviewer 2: ```suggestion
- Devices that contribute to the network during a specific event, such as a natural disaster, may receive a boost in rewards.
```

> - Demand driven rewards, a third party who needs data from specific cells.

Reviewer 2: This is the cell boost.
```suggestion
```
